### ISSUE TITLE
MVP

### ISSUE THREAD

**Issue by clincha** [clincha/StarSheet#3]
Google Sheets is a limited front end for this data. A better front end would be something like Kibana from the Elastic Stack. I could also use Elastic Search to store the data in document format for parsing.

Create an MVP using the existing Starling API calls to put data into an Elastic instance. Visualise the data in an interesting way.

**Comment by clincha** [comment]
I've created the MVP. It wasn't too difficult and this is the result:

![image](https://user-images.githubusercontent.com/10735762/212460411-3bbca1d0-95b5-4031-9923-3477ef27a58e.png)

It took about 20 lines of modified code to get the API passing data into Elastic Search. I used the [Elastic cloud offering](https://cloud.elastic.co/) to host the instances which made it really easy to get something up and running quickly.
